### PR TITLE
[2.4] Migrate resource models to the new app

### PR DESCRIPTION
**Pull request by matti-lamppu**
## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- See title.
- Old migrations will remain in place until everything is moved to not break the django migration system.
- Should follow [this guide](https://realpython.com/move-django-model/#the-short-way-reference-the-new-django-model-to-the-old-table) (the short way).

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of the fastest way to test your changes."

- Automated tests

## 🚧 Dependencies
[//]: # "Is this PR dependent on other changes outside this repository? Describe the changes, or add links to them."
[//]: # "e.g. This PR breaks X and is waiting for frontend changes before merging."

- None

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)."

- [TILA-3457](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3457)

[TILA-3457]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3457?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: locale/fi/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -579,8 +580,9 @@ msgid "Reservation purpose"
 msgstr "Varaustarkoitukset"
 
 #: applications/admin/application_section/form.py
-#: merchants/admin/payment_order.py reservation_units/models/introduction.py
+#: reservation_units/models/introduction.py
 #: reservation_units/models/reservation_unit_pricing.py
+#: tilavarauspalvelu/admin/payment_product/admin.py
```
Comment (vergama) [inline locale/fi/LC_MESSAGES/django.po:585]: cf. https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/1343#discussion_r1766203958

File: locale/fi/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3077,6 +2895,126 @@ msgstr "Ei, vie minut takaisin"
...
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Merchant name"
+msgstr "Kauppiaan nimi"
+
+#: tilavarauspalvelu/admin/payment_merchant/admin.py
+msgid "Value comes from the Merchant Experience API"
+msgstr "Arvo tulee kauppiaskokemusrajapinnasta"
+
+#: tilavarauspalvelu/admin/payment_product/admin.py
```
Comment (vergama) [inline locale/fi/LC_MESSAGES/django.po:2938]: cf. https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/1343#discussion_r1766203958
(and multiple lines immediately below)

File: locale/fi/LC_MESSAGES/django.po (hunk 1 above)
Comment (matti-lamppu) [reply locale/fi/LC_MESSAGES/django.po]: Fixed.

File: locale/fi/LC_MESSAGES/django.po (hunk 2 above)
Comment (matti-lamppu) [reply locale/fi/LC_MESSAGES/django.po]: Fixed

**Review by vergama — APPROVED** [review]

File: locale/sv/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -2217,12 +2217,16 @@ msgid "Keyword group"
...
 msgid "Reservation Unit"
-msgstr ""
+msgstr "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 #: reservation_units/models/reservation_unit.py
+#, fuzzy
+#| msgid "Reservation for %(name)s"
 msgid "Reservation Units"
-msgstr ""
+msgstr "Plural-Forms: nplurals=2; plural=(n != 1);\n"
```
Comment (ranta) [inline locale/sv/LC_MESSAGES/django.po:2229]: Fuzzy translations

**Pull request merged**